### DISCUSSION TITLE
platform does not match error on raspberry pi 4

### DISCUSSION THREAD

**Discussion by CL1302** [nextcloud/all-in-one#1230] [Questions] [answered]
Hi there
Thanks for this project to all supporter!

I always receive the following error while trying to start the docker container on my Raspberry Pi 4:
`WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/arm/v7) and no specific platform was requested.`

The container is built, but unhealthy. Without any logs, published ports or network.

System Info:
Linux 5.15.72-v8+ #1591 SMP PREEMPT Wed Oct 5 12:07:56 BST 2022 aarch64 GNU/Linux

I tried the following command:
```
# For arm64 CPUs:
sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest-arm64
```

as well as some docker-compose configurations.

it seems that the container restarts all the time:
nextcloud/all-in-one:latest-arm64   "start.sh /usr/bin/s…"   5 minutes ago   Restarting (159) 49 seconds ago  nextcloud-aio-mastercontainer

any idea??
Thanks
Chris

**Comment by szaimen** [comment, marked as answer]
Hi, are you sure that your pi 4 is running a 64-bit OS?

**Reply by CL1302** [reply]
Thank you szaimen
Indeed.  I thought  "..aarch64 GNU/Linux" means that, but it's only the kernel, not necessary the OS.

I will check that, it will probably be the problem....

Thanks!
Chris
